HW8_Airport/Air_Simulation/Baggage.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Air_Simulation
+{
+    class Baggage
+    {
+        public int weight;
+        private string[] forbiddenBag = new string[] { "drugs", "alcohol", "gasSpray", "ammunition", "toxicSubstance", "sharpObject" };
+        public Baggage(int weight)
+        {
+            if (weight > 20)
```

Comment:
20 - what a magic number? :) as a new developer will be involved in your project will not understand what this number mean. Have you got I mean there?